published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
+* [**js-ipfs**](//github.com/ipfs/js-ipfs) 
+	- Thanks to @diasdavid, `object features` now works, with tests and all. 
+	- As well, thanks to [@dignifiedquire](//github.com/dignifiedquire)'s efforts, the js-ipfs API now returns promises if there isn't a specified callback, allowing both major methods used by the Javascript community to work equally well.
+* [**registry-mirror**](//github.com/diasdavid/registry-mirror) 
+	- [@diasdavid](//github.com/diasdavid) merged registry-static into registry-mirror and removed all of the unnecessary stuff [(this was huge)](https://github.com/diasdavid/registry-mirror/pull/33).
+* [**station**](//github.com/ipfs/station)
+  - [@dignifiedquire](//github.com/dignifiedquire) fixed drag and drop
+    file uploading, and some dependency issues. So go ahead and try it
+    out.
+* [**ipfs-geoip**](//github.com/ipfs/ipfs-geoip)
+  - [@dignifiedquire](//github.com/dignifiedquire) rewrote the generation
+    script and cleaned up the code so that now the data is consistently
+    reproducable and fully stored on IPFS. This ensures that geoip lookups
+    over IFPS will work in all future releases.
+* [**fs-repo-migrations**](//github.com/ipfs/fs-repo-migrations)
+  - [@chriscool](//github.com/chriscool) Improved fs-repo-migrations tests by checking that the objects are the same and checking that garbage collection does not change anything. He also cleaned up sharness tests.
+* [**http-api-spec**](//github.com/ipfs/http-api-spec)
+  - @RichardLitt finished logging all existing `ipfs` commands for the HTTP API Spec. This means that if you have any questions about how the HTTP API should work, you can see them either in master or as open PRs to that issue. One step closer to a fully functional API...
+* [**community**](//github.com/ipfs/community)
+  - @RichardLitt proposed using the new [GitHub Templates]() for IPFS. What do you think about this idea? Chime in [here](https://github.com/ipfs/community/issues/108).
+  - We now have [a list of community events](https://github.com/ipfs/community#events-calendar) for IPFS on the ipfs/community README. Do you have any that you'd like to add? Check out past events, too.
+* [**ipfs-hyperlog**](//github.com/noffle/ipfs-hyperlog) - @noffle launched am IPFS-compatible Merkle DAG that replicates based on scuttlebutt logs and causal linking. `ipfs-hyperlog` is a drop-in replacement for [@mafintosh](https://github.com/mafintosh)'s [`hyperlog`](https://github.com/mafintosh/hyperlog). Its key difference is that it creates a Merkle DAG that is *binary compatible* with IPFS objects. This means any node of any DAG built using ipfs-hyperlog can be replicated to and from the IPFS network as well!
```

Comment:
maybe emphasize at the beginning that this is a fork of the project? it makes sound like this module was new. maybe:

- @noffle built ipfs-hyperlog, and ipfs-compatible version of hyperlog, a dag that ...